packages/uploadthing/src/tw.ts

```diff
@@ -10,12 +10,29 @@ const PACKAGES = ["react", "solid", "svelte", "vue"];
 export function withUt(twConfig: Config) {
   const contentPaths = PACKAGES.map((pkg) => {
     try {
-      const resolved = require.resolve(`@uploadthing/${pkg}`);
-      return resolved.split(sep).slice(0, -1).join(sep) + `${sep}**`;
+      const resolved = require.resolve(`@uploadthing/${pkg}`, {
+        paths: [...module.paths, process.cwd()],
```

Comment:
Thank you for this, I probably would have spent quite some time to think of trying this xd